packages/marshal/docs/smallcaps-cheatsheet.md

```diff
@@ -0,0 +1,29 @@
+# Smallcaps Cheatsheet
+
+An example-based summary of the Smallcaps encoding
+
+| Passable value  | OCapn name    | JS example          | JSON encoding        |
+| ----------------|---------------|---------------------|----------------------|
+| bigint          | SignedInteger | `7n`<br>`-7n`       | `"+7"`<br>`"-7"`     |
+| manifest constant | Undefined<br><br>Float64<br><br><br> | `undefined`<br>`Infinity`<br>`-Infinity`<br>`NaN`<br>`-0` | `"#undefined"`<br>`"#Infinity"`<br>`"#-Infinity"`<br>`"#NaN"`<br>`"#-0"` // unimplemented |
+| passable symbol | Symbol        | `Symbol.for('foo')`<br>`Symbol.asyncIterator` | `"%foo"`<br>`"%@@asyncIterator"` |
+| remotable       | Capability    | `Far('foo', {})`    | `"$0.foo"`           |
+| promise         | Capability    | `Promise.resolve()` | `"&1"`               |
+| special string  | String        | `'#foo'`            | `"!#foo"`            |
+| other string    | String        | `'foo'`             | `"foo"`              |
+| other JSON scalar | Null<br>Boolean<br><br>Float64 | `null`<br>`true`<br>`false`<br>`7.1` | `null`<br>`true`<br>`false`<br>`7.1` |
+| copyArray       | Sequence      | `[a,b]`             | `[<a>,<b>]`          |
+| copyRecord      | Struct        | `{x:a,y:b}`         | `{<x>:<a>,<y>:<b>}`  |
+| error           | Error         | `TypeError(msg)`    | `{"#error":<msg>,"name":"TypeError"}` |
```

Comment:
It might be illustrative to make the message concrete and show how or whether it gets escaped.

Reply:
I assume our handling of other properties is tentative, but it might be useful to illustrate how we handle a collision between the struct field name `#error` and the special field names `#error` or `#tag`.

Reply:
> It might be illustrative to make the message concrete and show how or whether it gets escaped.

For this doc, my inclination is to let the angle brackets do the work.

> I assume our handling of other properties is tentative, 

Added two bullets to clarify that we expect errors to grow more optional properties over time.

> but it might be useful to illustrate how we handle a collision between the struct field name #error and the special field names #error or #tag.

Again, I think the angle brackets already do all that work.